ua/lpnu/students/labs/first/hotel/Hotel.java

```diff
@@ -0,0 +1,55 @@
+package ua.lpnu.students.labs.first.hotel;
+
+public record Hotel(
+    String name,
+    long numOfVisitorsPerYear,
+    int numOfRooms,
+    HotelStatus status,
+    boolean hasDinner,
+    String adress
+){
+    public Hotel{
+        Hotel.numOfHotels++;
+    }
+
+    public Hotel(final String name, final int numOfRooms, final String adress){
+        this(name, -1, numOfRooms, HotelStatus.MIDDLE_PRICE, false, adress);
+    }
+
+    public Hotel(){
+        this(noneInfMessage, -1, noneInfMessage);
+    }
+
+    @Override
+    public String toString(){
+        return 
+            name + "\n\t" + 
+            visitorsMessage + ((numOfVisitorsPerYear == -1)? noneInfMessage: numOfVisitorsPerYear) +"\n\t"+
+            roomsMessage + ((numOfRooms == -1)? noneInfMessage: numOfRooms) +"\n\t"+
+            statusMessage + status.toString() +"\n\t"+
+            dinnerMessages[((hasDinner == true)?0:1)]+"\n\t"+
+            adressMessage + adress
```

Comment:
Consider using `String.format` for this whole string.

Reply:
Did that in the last commit